plugins/catalog-backend/package.json

```diff
@@ -12,7 +12,8 @@
     "test": "backstage-cli test",
     "prepack": "backstage-cli prepack",
     "postpack": "backstage-cli postpack",
-    "clean": "backstage-cli clean"
+    "clean": "backstage-cli clean",
+    "mock-db": "sh ./scripts/mock-db"
```

Comment:
mock-data?

i mean the fact that there's a db underneath is an implementation detail :)

Reply:
Yep. Makes sense.